streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetricsRecorder.java

```diff
@@ -56,6 +62,9 @@ public void maybeCloseStatistics() {
         }
     }
 
+    private static final String ROCKSDB_PROPERTIES_PREFIX = "rocksdb.";
+    private static final ByteBuffer CONVERSION_BUFFER = ByteBuffer.allocate(Long.BYTES);
```

Comment:
It seems a little risky to use this in a multithreaded context. Why not just create a new short-lived buffer each time for the conversion?

Reply:
Good point! I missed that the gauge can be called by multiple metrics reporters concurrently.